packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -587,6 +591,113 @@ it('should not show sort icon if disabled in column level', () => {
   expect(container.querySelector('.iui-sort .iui-icon-wrapper')).toBeFalsy();
 });
 
+it('should correct sort icons for ascending first on hover and click', async () => {
+  const mockedColumns = [
+    {
+      Header: 'Header name',
+      columns: [
+        {
+          id: 'name',
+          Header: 'Name',
+          accessor: 'name',
+        },
+      ],
+    },
+  ];
+  const { container } = renderComponent({
+    columns: mockedColumns,
+    isSortable: true,
+  });
+  const {
+    container: { firstChild: sortUpIcon },
+  } = render(<SvgSortUp className='iui-icon iui-sort' aria-hidden />);
+  const {
+    container: { firstChild: sortDownIcon },
+  } = render(<SvgSortDown className='iui-icon iui-sort' aria-hidden />);
+  const nameHeader = container.querySelector(
+    '.iui-table-header .iui-cell',
+  ) as HTMLDivElement;
+  expect(nameHeader).toBeTruthy();
+
+  // hover on column header
+  fireEvent.mouseEnter(nameHeader);
```

Comment:
this hover does nothing actually, because you query DOM and the icon is always there.
We could also check if icon is visible on hover (with `toBeVisible`), but for some reason it did not really work for me as that icon is visible even without hover (test passes if I do not hover and expect it to be visible) :/

Reply:
Removed `mouseEnter`